Manifest.toml

```diff
@@ -1,330 +1,347 @@
 # This file is machine-generated - editing it directly is not advised
 
```

Comment:
is there an advantage in changing the Manifest,? maybe I should never have included it? perhaps not include in your PR?

Reply:
It was suggested by julia 1.7 but i'll revert it anyways!